### PR TITLE
Multistage v2 recipe for beta

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: messaging-experiments.json [hunk 1]
```diff
@@ -1116,5 +1116,279 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V2-79-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '79.' && (env.channel == 'default' || env.channel == 'beta')",
```
Comment (punamdahiya) [inline messaging-experiments.json:1123]: filter expression intentionally allows both beta and default for easier testing on local

File: messaging-experiments.json [hunk 2]
```diff
@@ -1116,5 +1116,279 @@
...
+    "targeting": "localeLanguageCode == \"en\" && [userId, \"message-multi-stage-aboutwelcome-v2-79\"]|bucketSample(0, 2000, 10000) && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'message-multi-stage-aboutwelcome-v2-79' in activeExperiments)",
+    "arguments": {
+      "slug": "message-multi-stage-aboutwelcome-v2-79",
```
Comment (punamdahiya) [inline messaging-experiments.json:1126]: name space and slug should be updated with values from experimenter after signoffs in experimenter

File: messaging-experiments.json [hunk 3]
```diff
@@ -1116,5 +1116,279 @@
...
+                    }
+                  }
+                }
+              },
+              {
+                "id": "AW_IMPORT_SETTINGS",
+                "order": 1,
+                "content": {
+                  "zap": true,
+                  "title": "Import your passwords, bookmarks,\u00a0and more",
```
Comment (ncloudioj) [inline messaging-experiments.json:1185]: Interesting, the YAML version doesn't show the unicode non-breaking space. Can you double check?

**Comment by punamdahiya** [comment]
> Looks good.
> 
> Somehow the CI didn't trigger the validation check, could you run it locally? (via `make check`?)

yes it passes locally, thanks!

File: messaging-experiments.json (hunk 3 above)
Comment (punamdahiya) [reply messaging-experiments.json:1185]: yes, it do show up in json, so all good

**Pull request merged**